### ISSUE TITLE
Include vendor deprecation warnings

### ISSUE THREAD

**Issue by arderyp** [sensiolabs-de/deprecation-detector#124]
Firstly, thanks for the awesome tool!

I've run it against my codebase with:

```
./deprecation-detector check $PROJECT/src $PROJECT/vendor
```

It found lots of helpful things to cleanup.  Now when I run it, nothing is found.  However, if I go to one of my views, I do see deprecation warnings in the profiler related to my Twig dependency.  It's not my issue, but the related deprecation is discussed here: https://github.com/twigphp/Twig/issues/2267

If there is any more info I can provide that would be helpful, please let me know.

**Comment by slde-robin** [comment]
Actually the tool doesn't check if there are any deprecations in your twig templates.

**Comment by arderyp** [comment]
Right, that's the reason I opened the issue, so there could be some discussion about it.

Do you think this type of functionality could be added?  I'm thinking beyond templates specifically.  Do you think the tool could find and report all deprecation warnings that the symfony profiler would report? That would be the ultimate goal, I think.

**Comment by slde-robin** [comment]
The profiler gathers the warnings by actually executing the code.
The detector does not execute the code.

To add Twig support we would need a twig parser and so on... IMO thats too much.

But you could check your compiled Twig templates in the cache dir for deprecations. Right now I don't have the time to test that myself.

**Comment by arderyp** [comment]
Awesome, thanks for the very helpful info.
